gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -367,21 +367,26 @@
   /**
    * Common metrics configuration properties.
    */
-  public static final String METRICS_LOG_DIR_KEY = "metrics.log.dir";
-  public static final String METRICS_ENABLED_KEY = "metrics.enabled";
+  public static final String METRICS_CONFIGURATIONS_PREFIX = "metrics.";
+  public static final String METRICS_LOG_DIR_KEY = METRICS_CONFIGURATIONS_PREFIX + "log.dir";
+  public static final String METRICS_ENABLED_KEY = METRICS_CONFIGURATIONS_PREFIX + "enabled";
+  public static final String METRICS_FILE_SUFFIX = METRICS_CONFIGURATIONS_PREFIX + "reporting.file.suffix";
```

Comment:
Metrics enabled already defaults to false, and metrics.reporting.file.suffix defaults to empty, although it is not in this file (I will add it here). I don't think giving a default to metrics.log.dir is a good idea, we could be messing up with the user's file system without them realizing it. On the other hand, metrics.reporting.file.enabled should probably be set to false.